lib/zones/route_data.js

```diff
@@ -17,6 +17,7 @@ module.exports = function(data){
 				oldData = can.route.data;
 				intercept(can.route, "data", function(data){
 					viewModel = data;
+					viewModel.bind("statusCode", noop);
```

Comment:
It does run on  a set, it's so you can do (in your main):

```js
module.exports = function(){
  var vm = new VM();
  can.route.map(vm);

  ...
};
```

This doesn't run in the browser.  It only runs this when the Zone is created, not on every task. Binding when the Zone gets created, unbind when it ends.